react/week4/managing-state/src/TodoListItems.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const TodoListItems=(props) => {    
+  const details = <div>
+                    <span>{props.todoitem.login}</span>  
+                    <img src={props.todoitem.avatar_url} alt={props.todoitem.avatar_url}></img>
+                  </div>
+  return <li>
+            <div>
+              {details}
+            </div>
+          </li>;
+}
```

Comment:
I think storing half of your component in a variable is a bit of an unnecessary abstraction. I always try to keep things as simple and readable as possible: 
```suggestion
const TodoListItems=(props) => {    
  return <li>
            <div>
                    <span>{props.todoitem.login}</span>  
                    <img src={props.todoitem.avatar_url} alt={props.todoitem.avatar_url}></img>
            </div>
          </li>;
}
```

Reply:
Yes, make sense